unstructured/partition/pdf.py

```diff
@@ -322,18 +329,19 @@ def _process_pdfminer_pages(
                     element.metadata = metadata
                     page_elements.append(element)
 
+        if include_page_breaks:
+            elements.append(PageBreak())
+
```

Comment:
Do we not want the page break to be appended after the content of the page is sorted? Otherwise this can be modified from being at the end of the page.

Reply:
You're right, updated.